app/routes/account.js

```diff
@@ -1,34 +1,7 @@
 import Route from '@ember/routing/route';
-import RSVP from 'rsvp'
 
 export default Route.extend({
   model({account_id}){
```

Comment:
I prefer aliasing non-conventional var names for eg `account_id: accountId` to keep var names convention.